include/ringmesh/basic/types.h

```diff
@@ -54,12 +54,18 @@ namespace RINGMesh
     // Using definitions of Geogram/basic/numeric.h
     using GEO::Numeric::float32;
     using GEO::Numeric::float64;
+    using GEO::Numeric::uint8;
+    using GEO::Numeric::int8;
+    using GEO::Numeric::uint32;
+    using GEO::Numeric::int32;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this using-declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1003)